src/utils/requests.ts

```diff
@@ -0,0 +1,36 @@
+const NAV_LINKS = {
+  docs: 'https://docs.goosefx.io/',
+  blog: 'https://www.blog.goosefx.io/',
+  whatsnew: 'todo',
+  twitter: 'https://twitter.com/GooseFX1',
+  discord: 'https://discord.com/invite/cDEPXpY26q',
+  telegram: 'https://t.me/goosefx',
+  terms: 'https://www.goosefx.io/terms',
+  risks: 'https://www.goosefx.io/risks-and-disclaimers',
+  securityAudit: 'https://media.goosefx.io/GFX_Ottersec_Audit.pdf',
+  securityAuditer: 'https://osec.io/'
+} as const
+
+function navigateTo(src: string, target = '_self', features = '') {
+  return (): Window => window.open(src, target, features)
```

Comment:
@deadeagle63 - this navigation is not working. i am not able to see these hyperlinks open

Reply:
Fixed, this returns a curried function so we can bind to buttons. if no other requirement is needed we can probably remove this and just use a tags if we can get some designs in :)